R/combinedscatterplot.R

```diff
@@ -329,9 +331,8 @@ CombinedScatter <- function(x = NULL,
     labels.or.logos <- if (!is.null(logo.urls)) logo.urls else scatter.labels
     logo.size <- rep(logo.size, n)
 
-    footer <- processFooter(footer, scatter.labels.name, scatter.colors.name,
```

Comment:
Are we no longer automatically filling in the footer? Or will it be done elsewhere?

Reply:
No longer automatically filling in the footer. Because the information there is already in the legends so its not needed